library/src/main/java/software/amazon/location/auth/data/response/GetCredentialResponse.kt

```diff
@@ -0,0 +1,16 @@
+package software.amazon.location.auth.data.response
+
+import com.google.gson.annotations.SerializedName
+
+
+data class Credentials(
```

Comment:
move each class to a separate file with the same name, one file = one class

Reply:
Done